ts/generated/ffi.WASM_DEBUG_ASYNCIFY.ts

```diff
@@ -161,7 +161,7 @@ export class QuickJSAsyncFFI {
   QTS_BuildIsAsyncify: () => number =
     this.module.cwrap("QTS_BuildIsAsyncify", "number", [])
 
-  QTS_NewFunction: (ctx: JSContextPointer, func_id: number, name: string) => JSValuePointer =
+  QTS_NewFunction: (ctx: JSContextPointer, func_id: uint32_t, name: string) => JSValuePointer =
```

Comment:
is uint32_t valid typescript?

Reply:
no, this needs to be mapped in the generator. Working on that part now